src/Repository/ProjectsRepository.php

```diff
@@ -19,6 +19,18 @@ public function __construct(ManagerRegistry $registry)
         parent::__construct($registry, Projects::class);
     }
 
+//    public function getProjectsByTeamId ($id)
+//    {
+//        $query = $this->createQueryBuilder('projects');
+//        $query
+//            ->select('projects')
+//        ->where($query->expr()->orX($query->expr()->eq('projects.id', $id)));
+//
+//
+//
+//        return $query->getQuery()->getArrayResult();
+//    }
+
```

Comment:
if you are commenting something, please remove before you pushing into to the develop or master